### PR TITLE
minor issues resolved

### PR DESCRIPTION
**Pull request by cq-osaf-ali**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 const check = require("./components/checkUser");
 const saveImg = require("./components/saveImage");
+const userLogs = require("./components/userLogs");
 
 const wssClient = new WebSocket({ port: 9000 });
 wssClient.on("connection", (websocket) => {
   console.log("connection opened on port 9000");
 });
+
 const wss = new WebSocket({ port: 8080 });
```
Comment (cq-ubaid-khan) [inline src/app.js:15]: specify ports in config and get from there.

File: src/Database/db.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const db = require("knex")({
+  client: "pg",
+  connection: {
+    user: "postgres",
+    password: "cq123",
+    host: "localhost",
+    port: 5432,
+    database: "employee",
+  },
```
Comment (cq-ubaid-khan) [inline src/Database/db.js:9]: move all the db configuration to config file and get from there, define an object there called db
```
db: {
   client: 'pg',
   user: ...
   password: ...
   .....
}

```
and here 
```
const db = config.db;
{
  client: db.client,
  connection: { 
      user: db.user,
      .......
   }
}
```

File: src/app.js [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
+open(`${__dirname}\\public\\index.html`);
 wss.on("connection", (ws) => {
-  // let number;
   let userName;
   let isClosed = false;
+  const sessionStart = new Date().toISOString();
+  userLogs.logInTime(sessionStart);
   ws.on("message", (message) => {
     const data = JSON.parse(message);
     console.log(data.name);
```
Comment (cq-ubaid-khan) [inline src/app.js:24]: at least add one line spacing between blocks, and block and other variables

File: src/components/userLogs.js [hunk 4]
```diff
@@ -7,25 +9,45 @@ const logTime = {
...
+function saveUser(userName, startSession, endSession) {
+  let userId = null;
+  db.insert({ user_login: userName })
+    .into("users").returning("uuid").then((id) => {
+      console.log(id);
+      userId = parseInt(id, 10);
+      saveSession(startSession, endSession, userId);
+    });
+}
+
```
Comment (cq-ubaid-khan) [inline src/components/userLogs.js:41]: create dao files and move all db interaction logic, like CRUD in there. do not write db code in components

File: src/app.js (hunk 3 above)
Comment (cq-osaf-ali) [reply src/app.js:24]: Done

File: src/components/userLogs.js (hunk 4 above)
Comment (cq-osaf-ali) [reply src/components/userLogs.js:41]: done

File: src/Database/db.js (hunk 2 above)
Comment (cq-osaf-ali) [reply src/Database/db.js:9]: done

File: src/app.js (hunk 1 above)
Comment (cq-osaf-ali) [reply src/app.js:15]: done

**Pull request merged**